Frontend/android-native/mobility-app/src/main/java/in/juspay/mobility/app/MobilityAppBridge.java

```diff
@@ -146,6 +146,17 @@ public MobilityAppBridge(BridgeComponents bridgeComponents) {
         clevertapDefaultInstance = CleverTapAPI.getDefaultInstance(bridgeComponents.getContext());
     }
 
+    @JavascriptInterface
+    public void restartApp() {
```

Comment:
After selecting city, app will be restarted?

Reply:
Yes.

Reply:
This is not good as per the good practices, @tejkranthi do we want whole app to be restarted? why is it required?

Reply:
No. We don't want the app restarted. The city specific splash screen must show up from next app launch. For new user, as soon as city is confirmed, it will only show a welcome screen and move on. Not restart the app.

Reply:
Thanks for confirming @tejkranthi , @Vignesh-772 Designers might have just wanted a splash of screen with regional content but not app restart we can see if we can build something similar experience in just PS later without restarting the app. For now we can skip this restarting part